tutorials/scripting/cross_language_scripting.rst

```diff
@@ -171,6 +171,30 @@ If that's impossible, you'll see the following error: ``Invalid call. Nonexisten
     my_csharp_node.PrintArray(["a", "b", "c"]) # a, b, c
     my_csharp_node.PrintArray([1, 2, 3]) # 1, 2, 3
 
+Receiving C# collections from GDScript
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example:
+
+.. code-block:: csharp
+
+    // These methods will not be visible to GDScript, and Godot will throw the "Invalid call" error
+    public List<string> GetStrings() { ... }
```

Comment:
```suggestion
    public System.Collections.Generic.List<string> GetStrings() { ... }
```

I'd be explicit here, given that this article is for both GDScript and C# users.